itest/integration_test.go

```diff
@@ -55,9 +55,22 @@ func (suite *IntegrationTestSuite) SetupSuite() {
 func (suite *IntegrationTestSuite) TestConnect() {
```

Comment:
Should we rename this test to `testConnectWithString()` or `testConnectWithDsn()`?